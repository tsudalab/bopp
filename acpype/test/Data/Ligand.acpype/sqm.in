Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0002,
  tight_p_conv=1, scfconv=1.d-10,   qmcharge=0,
 /
   6    C1        1.2870       -5.8370       -5.4640 
   6    C3        1.2510       -3.5270       -6.4580 
   6    C4        0.7870       -2.6580       -5.2880 
   6    C5        1.4310       -2.9890       -3.9630 
   6    C6        1.1020       -4.3790       -3.4210 
   6   C20        0.1740       -5.5520       -7.5220 
   6   C21       -1.0010       -4.8190       -8.0300 
   6   C22       -1.3880       -4.9450       -9.3530 
   6   C23       -2.4920       -4.2510       -9.8210 
   6   C24       -3.2070       -3.4310       -8.9630 
   6   C25       -2.8200       -3.3060       -7.6430 
   6   C26       -1.7180       -3.9980       -7.1760 
   6   C27       -4.4030       -2.6760       -9.4620 
   6   C30        2.7300       -3.4750       -6.8310 
   6   C31        2.9470       -3.7610       -8.2940 
   6   C32        3.9260       -4.6640       -8.6660 
   6   C33        4.1310       -4.9530       -9.9980 
   6   C34        3.3580       -4.3390      -10.9630 
   6   C35        2.3770       -3.4320      -10.5960 
   6   C36        2.1690       -3.1440       -9.2580 
   6   C60       -0.3610       -4.6870       -3.1140 
   6   C61       -0.4980       -5.7060       -2.0130 
   6   C62       -1.3450       -6.7830       -2.1920 
   6   C63       -1.4740       -7.7370       -1.2060 
   6   C64       -0.7570       -7.6160       -0.0340 
   6   C65        0.0930       -6.5370        0.1540 
   6   C66        0.2240       -5.5810       -0.8390 
   6   C70        2.4300       -6.5300       -3.5260 
   6   C71        3.5390       -6.0350       -2.6890 
   6   C72        3.9820       -6.7700       -1.6010 
   6   C73        5.0280       -6.2950       -0.8260 
   6   C74        5.6280       -5.0860       -1.1390 
   6   C75        5.1850       -4.3530       -2.2240 
   6   C76        4.1420       -4.8260       -2.9990 
   6   C77        6.7610       -4.5680       -0.3030 
   7    N2        0.8710       -4.9570       -6.3850 
   7    N7        1.6330       -5.5160       -4.2100 
   8    O1        1.3580       -7.0130       -5.7920 
   8    O4        1.0440       -1.3470       -5.6430 
   8    O5        1.0440       -2.0840       -2.9920 
   8   O27       -4.0310       -1.8430      -10.5500 
   8   O77        6.2750       -4.1440        0.9620 
   1    H3        0.7200       -3.1370       -7.3170 
   1    H4       -0.2800       -2.7800       -5.1810 
   1    H5        2.5020       -2.9230       -4.0840 
   1    H6        1.6160       -4.4310       -2.4690 
   1  H201       -0.1370       -6.5490       -7.2360 
   1  H202        0.8780       -5.6440       -8.3330 
   1   H22       -0.8320       -5.5830      -10.0200 
   1   H23       -2.7950       -4.3490      -10.8530 
   1   H25       -3.3780       -2.6660       -6.9760 
   1   H26       -1.4170       -3.8980       -6.1430 
   1  H271       -4.8010       -2.0740       -8.6550 
   1  H272       -5.1570       -3.3720       -9.7910 
   1  H301        3.2560       -4.2130       -6.2450 
   1  H302        3.1200       -2.4950       -6.6010 
   1   H32        4.5300       -5.1420       -7.9130 
   1   H33        4.8960       -5.6580      -10.2860 
   1   H34        3.5190       -4.5700      -12.0000 
   1   H35        1.7730       -2.9520      -11.3520 
   1   H36        1.4060       -2.4380       -8.9680 
   1  H601       -0.8250       -5.0720       -4.0100 
   1  H602       -0.8620       -3.7780       -2.8180 
   1   H62       -1.9060       -6.8780       -3.1050 
   1   H63       -2.1360       -8.5770       -1.3510 
   1   H64       -0.8580       -8.3630        0.7290 
   1   H65        0.6540       -6.4440        1.0720 
   1   H66        0.8840       -4.7390       -0.6960 
   1  H701        2.8340       -7.1970       -4.2770 
   1  H702        1.7710       -7.1020       -2.8930 
   1   H72        3.5150       -7.7100       -1.3580 
   1   H73        5.3740       -6.8660        0.0220 
   1   H75        5.6540       -3.4110       -2.4670 
   1   H76        3.7980       -4.2510       -3.8470 
   1  H771        7.2270       -3.7370       -0.8170 
   1  H772        7.4890       -5.3490       -0.1590 
   1   HO4        0.6420       -1.1870       -6.5000 
   1   HO5        1.6930       -2.1110       -2.2860 
   1  HO27       -4.6740       -1.9660      -11.2520 
   1  HO77        6.4640       -4.8360        1.6000 

