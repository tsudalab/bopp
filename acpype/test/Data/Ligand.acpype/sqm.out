            --------------------------------------------------------
                             AMBER SQM VERSION 1.4
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                Tim Giese, Andreas W. Goetz and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =109

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.1D-09
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is disabled.
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.2870   -5.8370   -5.4640
  QMMM:     2        2      C        1.2510   -3.5270   -6.4580
  QMMM:     3        3      C        0.7870   -2.6580   -5.2880
  QMMM:     4        4      C        1.4310   -2.9890   -3.9630
  QMMM:     5        5      C        1.1020   -4.3790   -3.4210
  QMMM:     6        6      C        0.1740   -5.5520   -7.5220
  QMMM:     7        7      C       -1.0010   -4.8190   -8.0300
  QMMM:     8        8      C       -1.3880   -4.9450   -9.3530
  QMMM:     9        9      C       -2.4920   -4.2510   -9.8210
  QMMM:    10       10      C       -3.2070   -3.4310   -8.9630
  QMMM:    11       11      C       -2.8200   -3.3060   -7.6430
  QMMM:    12       12      C       -1.7180   -3.9980   -7.1760
  QMMM:    13       13      C       -4.4030   -2.6760   -9.4620
  QMMM:    14       14      C        2.7300   -3.4750   -6.8310
  QMMM:    15       15      C        2.9470   -3.7610   -8.2940
  QMMM:    16       16      C        3.9260   -4.6640   -8.6660
  QMMM:    17       17      C        4.1310   -4.9530   -9.9980
  QMMM:    18       18      C        3.3580   -4.3390  -10.9630
  QMMM:    19       19      C        2.3770   -3.4320  -10.5960
  QMMM:    20       20      C        2.1690   -3.1440   -9.2580
  QMMM:    21       21      C       -0.3610   -4.6870   -3.1140
  QMMM:    22       22      C       -0.4980   -5.7060   -2.0130
  QMMM:    23       23      C       -1.3450   -6.7830   -2.1920
  QMMM:    24       24      C       -1.4740   -7.7370   -1.2060
  QMMM:    25       25      C       -0.7570   -7.6160   -0.0340
  QMMM:    26       26      C        0.0930   -6.5370    0.1540
  QMMM:    27       27      C        0.2240   -5.5810   -0.8390
  QMMM:    28       28      C        2.4300   -6.5300   -3.5260
  QMMM:    29       29      C        3.5390   -6.0350   -2.6890
  QMMM:    30       30      C        3.9820   -6.7700   -1.6010
  QMMM:    31       31      C        5.0280   -6.2950   -0.8260
  QMMM:    32       32      C        5.6280   -5.0860   -1.1390
  QMMM:    33       33      C        5.1850   -4.3530   -2.2240
  QMMM:    34       34      C        4.1420   -4.8260   -2.9990
  QMMM:    35       35      C        6.7610   -4.5680   -0.3030
  QMMM:    36       36      N        0.8710   -4.9570   -6.3850
  QMMM:    37       37      N        1.6330   -5.5160   -4.2100
  QMMM:    38       38      O        1.3580   -7.0130   -5.7920
  QMMM:    39       39      O        1.0440   -1.3470   -5.6430
  QMMM:    40       40      O        1.0440   -2.0840   -2.9920
  QMMM:    41       41      O       -4.0310   -1.8430  -10.5500
  QMMM:    42       42      O        6.2750   -4.1440    0.9620
  QMMM:    43       43      H        0.7200   -3.1370   -7.3170
  QMMM:    44       44      H       -0.2800   -2.7800   -5.1810
  QMMM:    45       45      H        2.5020   -2.9230   -4.0840
  QMMM:    46       46      H        1.6160   -4.4310   -2.4690
  QMMM:    47       47      H       -0.1370   -6.5490   -7.2360
  QMMM:    48       48      H        0.8780   -5.6440   -8.3330
  QMMM:    49       49      H       -0.8320   -5.5830  -10.0200
  QMMM:    50       50      H       -2.7950   -4.3490  -10.8530
  QMMM:    51       51      H       -3.3780   -2.6660   -6.9760
  QMMM:    52       52      H       -1.4170   -3.8980   -6.1430
  QMMM:    53       53      H       -4.8010   -2.0740   -8.6550
  QMMM:    54       54      H       -5.1570   -3.3720   -9.7910
  QMMM:    55       55      H        3.2560   -4.2130   -6.2450
  QMMM:    56       56      H        3.1200   -2.4950   -6.6010
  QMMM:    57       57      H        4.5300   -5.1420   -7.9130
  QMMM:    58       58      H        4.8960   -5.6580  -10.2860
  QMMM:    59       59      H        3.5190   -4.5700  -12.0000
  QMMM:    60       60      H        1.7730   -2.9520  -11.3520
  QMMM:    61       61      H        1.4060   -2.4380   -8.9680
  QMMM:    62       62      H       -0.8250   -5.0720   -4.0100
  QMMM:    63       63      H       -0.8620   -3.7780   -2.8180
  QMMM:    64       64      H       -1.9060   -6.8780   -3.1050
  QMMM:    65       65      H       -2.1360   -8.5770   -1.3510
  QMMM:    66       66      H       -0.8580   -8.3630    0.7290
  QMMM:    67       67      H        0.6540   -6.4440    1.0720
  QMMM:    68       68      H        0.8840   -4.7390   -0.6960
  QMMM:    69       69      H        2.8340   -7.1970   -4.2770
  QMMM:    70       70      H        1.7710   -7.1020   -2.8930
  QMMM:    71       71      H        3.5150   -7.7100   -1.3580
  QMMM:    72       72      H        5.3740   -6.8660    0.0220
  QMMM:    73       73      H        5.6540   -3.4110   -2.4670
  QMMM:    74       74      H        3.7980   -4.2510   -3.8470
  QMMM:    75       75      H        7.2270   -3.7370   -0.8170
  QMMM:    76       76      H        7.4890   -5.3490   -0.1590
  QMMM:    77       77      H        0.6420   -1.1870   -6.5000
  QMMM:    78       78      H        1.6930   -2.1110   -2.2860
  QMMM:    79       79      H       -4.6740   -1.9660  -11.2520
  QMMM:    80       80      H        6.4640   -4.8360    1.6000

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -111.8268 kcal/mol        3.1075 kcal/(mol*A)
xmin    20     -115.4125 kcal/mol        0.3414 kcal/(mol*A)
xmin    30     -117.8962 kcal/mol        0.4926 kcal/(mol*A)
xmin    40     -119.5523 kcal/mol        0.3861 kcal/(mol*A)
xmin    50     -119.7628 kcal/mol        0.1645 kcal/(mol*A)
xmin    60     -119.8998 kcal/mol        0.0925 kcal/(mol*A)
xmin    70     -120.0266 kcal/mol        0.1137 kcal/(mol*A)
xmin    80     -120.1181 kcal/mol        0.0546 kcal/(mol*A)
xmin    90     -120.1452 kcal/mol        0.0156 kcal/(mol*A)
xmin   100     -120.1466 kcal/mol        0.0011 kcal/(mol*A)
xmin   110     -120.1467 kcal/mol        0.0002 kcal/(mol*A)
xmin   110     -120.1467 kcal/mol        0.0001 kcal/(mol*A)
  ... geometry converged !

 Heat of formation   =       -120.14667201 kcal/mol  (       -5.20995065 eV)
 Total SCF energy    =    -161889.82275423 kcal/mol  (    -7020.06950064 eV)
 Electronic energy   =   -1697964.22233153 kcal/mol  (   -73629.25381950 eV)
 Core-core repulsion =    1536074.39957730 kcal/mol  (    66609.18431886 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C               0.384164
     2      C              -0.010489
     3      C              -0.009695
     4      C               0.023124
     5      C               0.052942
     6      C              -0.011658
     7      C              -0.083032
     8      C              -0.121358
     9      C              -0.130735
    10      C              -0.113987
    11      C              -0.115687
    12      C              -0.110943
    13      C               0.014338
    14      C              -0.156200
    15      C              -0.082853
    16      C              -0.128907
    17      C              -0.128705
    18      C              -0.130954
    19      C              -0.132255
    20      C              -0.129092
    21      C              -0.141572
    22      C              -0.073837
    23      C              -0.122978
    24      C              -0.128668
    25      C              -0.131728
    26      C              -0.129916
    27      C              -0.133055
    28      C               0.016872
    29      C              -0.101641
    30      C              -0.116814
    31      C              -0.130498
    32      C              -0.113547
    33      C              -0.112632
    34      C              -0.114923
    35      C               0.014693
    36      N              -0.297019
    37      N              -0.314981
    38      O              -0.408140
    39      O              -0.338978
    40      O              -0.333015
    41      O              -0.319441
    42      O              -0.320130
    43      H               0.110355
    44      H               0.103860
    45      H               0.099401
    46      H               0.120251
    47      H               0.124811
    48      H               0.092850
    49      H               0.139947
    50      H               0.128868
    51      H               0.137232
    52      H               0.141071
    53      H               0.112005
    54      H               0.073213
    55      H               0.104803
    56      H               0.116470
    57      H               0.153794
    58      H               0.134984
    59      H               0.131639
    60      H               0.131042
    61      H               0.130083
    62      H               0.104223
    63      H               0.117168
    64      H               0.132563
    65      H               0.131918
    66      H               0.131276
    67      H               0.131033
    68      H               0.136266
    69      H               0.133489
    70      H               0.100829
    71      H               0.133710
    72      H               0.128535
    73      H               0.138133
    74      H               0.151598
    75      H               0.111231
    76      H               0.071791
    77      H               0.212122
    78      H               0.224941
    79      H               0.197762
    80      H               0.198665
 Total Mulliken Charge =   -0.000000
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.4210   -5.7828   -5.1758
  QMMM:     2        2      C        1.1420   -3.5230   -6.1194
  QMMM:     3        3      C        0.6000   -2.6747   -4.9406
  QMMM:     4        4      C        1.2130   -3.0577   -3.5896
  QMMM:     5        5      C        0.8129   -4.4772   -3.1064
  QMMM:     6        6      C        0.2620   -5.5004   -7.2895
  QMMM:     7        7      C       -0.9961   -4.8938   -7.8412
  QMMM:     8        8      C       -0.9934   -4.3216   -9.1177
  QMMM:     9        9      C       -2.1697   -3.8039   -9.6544
  QMMM:    10       10      C       -3.3600   -3.8410   -8.9199
  QMMM:    11       11      C       -3.3601   -4.4088   -7.6424
  QMMM:    12       12      C       -2.1858   -4.9353   -7.1075
  QMMM:    13       13      C       -4.6228   -3.2965   -9.5069
  QMMM:    14       14      C        2.6676   -3.3402   -6.2445
  QMMM:    15       15      C        3.1685   -3.6282   -7.6154
  QMMM:    16       16      C        3.5682   -4.9237   -7.9678
  QMMM:    17       17      C        4.0683   -5.1813   -9.2430
  QMMM:    18       18      C        4.1738   -4.1501  -10.1766
  QMMM:    19       19      C        3.7766   -2.8587   -9.8315
  QMMM:    20       20      C        3.2771   -2.5976   -8.5565
  QMMM:    21       21      C       -0.7206   -4.6232   -3.0711
  QMMM:    22       22      C       -1.1552   -5.7288   -2.1753
  QMMM:    23       23      C       -1.6859   -6.9098   -2.7063
  QMMM:    24       24      C       -2.1101   -7.9334   -1.8597
  QMMM:    25       25      C       -2.0104   -7.7853   -0.4768
  QMMM:    26       26      C       -1.4858   -6.6087    0.0589
  QMMM:    27       27      C       -1.0608   -5.5848   -0.7849
  QMMM:    28       28      C        2.3148   -6.4408   -3.0027
  QMMM:    29       29      C        3.7163   -5.9550   -2.7580
  QMMM:    30       30      C        4.1264   -5.5916   -1.4722
  QMMM:    31       31      C        5.4385   -5.1806   -1.2481
  QMMM:    32       32      C        6.3516   -5.1193   -2.3064
  QMMM:    33       33      C        5.9377   -5.4777   -3.5931
  QMMM:    34       34      C        4.6275   -5.8951   -3.8176
  QMMM:    35       35      C        7.7621   -4.6921   -2.0547
  QMMM:    36       36      N        0.6706   -4.8973   -6.0246
  QMMM:    37       37      N        1.5059   -5.5392   -3.8030
  QMMM:    38       38      O        1.9560   -6.8132   -5.6586
  QMMM:    39       39      O        0.9432   -1.3068   -5.1369
  QMMM:    40       40      O        0.7472   -2.1760   -2.5760
  QMMM:    41       41      O       -4.5554   -1.9098   -9.7828
  QMMM:    42       42      O        7.8680   -3.3498   -1.6175
  QMMM:    43       43      H        0.6777   -3.0829   -7.0574
  QMMM:    44       44      H       -0.5219   -2.7687   -4.8897
  QMMM:    45       45      H        2.3355   -2.9740   -3.6551
  QMMM:    46       46      H        1.1609   -4.5141   -2.0234
  QMMM:    47       47      H        0.0749   -6.6001   -7.1090
  QMMM:    48       48      H        1.0921   -5.4205   -8.0526
  QMMM:    49       49      H       -0.0618   -4.2855   -9.7032
  QMMM:    50       50      H       -2.1633   -3.3670  -10.6644
  QMMM:    51       51      H       -4.2908   -4.4400   -7.0558
  QMMM:    52       52      H       -2.1913   -5.3835   -6.1015
  QMMM:    53       53      H       -5.4834   -3.3931   -8.7894
  QMMM:    54       54      H       -4.8665   -3.8563  -10.4528
  QMMM:    55       55      H        3.2027   -3.9965   -5.5051
  QMMM:    56       56      H        2.9026   -2.2732   -5.9753
  QMMM:    57       57      H        3.4769   -5.7423   -7.2327
  QMMM:    58       58      H        4.3816   -6.2012   -9.5110
  QMMM:    59       59      H        4.5719   -4.3545  -11.1811
  QMMM:    60       60      H        3.8618   -2.0425  -10.5638
  QMMM:    61       61      H        2.9743   -1.5753   -8.2849
  QMMM:    62       62      H       -1.1341   -4.7801   -4.1013
  QMMM:    63       63      H       -1.1300   -3.6493   -2.6820
  QMMM:    64       64      H       -1.7672   -7.0315   -3.7970
  QMMM:    65       65      H       -2.5243   -8.8590   -2.2855
  QMMM:    66       66      H       -2.3462   -8.5926    0.1899
  QMMM:    67       67      H       -1.4103   -6.4870    1.1494
  QMMM:    68       68      H       -0.6546   -4.6534   -0.3616
  QMMM:    69       69      H        2.3739   -7.4405   -3.5288
  QMMM:    70       70      H        1.7907   -6.6022   -2.0162
  QMMM:    71       71      H        3.4144   -5.6328   -0.6341
  QMMM:    72       72      H        5.7599   -4.9056   -0.2322
  QMMM:    73       73      H        6.6483   -5.4290   -4.4322
  QMMM:    74       74      H        4.2982   -6.1853   -4.8299
  QMMM:    75       75      H        8.3754   -4.7251   -2.9967
  QMMM:    76       76      H        8.2274   -5.3723   -1.2876
  QMMM:    77       77      H        0.4578   -1.0029   -5.9126
  QMMM:    78       78      H        0.9319   -1.2781   -2.8835
  QMMM:    79       79      H       -3.7617   -1.7560  -10.3092
  QMMM:    80       80      H        7.2413   -3.2288   -0.8938

           --------- Calculation Completed ----------

